servers/mlflowserver/models/elasticnet_wine/Makefile

```diff
@@ -14,12 +14,16 @@ train:
 
 collect:
 	cp metadata.yaml output/
-	cp conda.yaml output/
 	cd mlruns && find . -name "MLmodel" -type f -exec cp {} ../output \;
+	cd mlruns && find . -name "conda.yaml" -type f -exec cp {} ../output \;
 	cd mlruns && find . -name "model.pkl" -type f -exec cp {} ../output \;
+	cd output && \
+		conda env create --force -n elasticnet-wine-env -f conda.yaml && \
```

Comment:
instead of `-n elasticnet-wine-env` could we use environment in local folder?

Reply:
Do you mean using `-p ./env/elasticnet-wine-env`? That's a great point. I'll update it.